### PR TITLE
Verify content in migrated markdown files

### PR DESCRIPTION
**Pull request by jimj-procoretech**
Checking and touching each file and fixing links, headers, removing extraneous line feeds, etc. Also converted most tables to html for ease of editing going forward. I also added {:target="_blank" rel="noopener"} on links to external sites.

### CODE REVIEW

File: api_essentials/rate_limiting.md [hunk 1]
```diff
@@ -6,20 +6,34 @@ section_title: API Essentials
...
-By limiting the number of API requests that can be issued from a unique authentication token, we are able to offer a more reliable service by protecting our own system infrastructure from being negatively affected by abusive users or applications that are not operating in accordance with our terms and conditions.
+Usage of Procore's API is subject to rate limits. By limiting the number of API requests that can be issued from a unique authentication token, we are able to offer a more reliable service by protecting our own system infrastructure from being negatively affected by abusive users or applications that are not operating in accordance with our terms and conditions.
 
 ## Procore Rate Limits
 
-Procore API rate limits to 3,600 requests per hour.
-The rate limit resets every hour.
-There are three important response headers returned when making a request to the Procore API.
-
-| Header                 | Explanation                                                                     |
```
Comment (claudioprocore) [inline api_essentials/rate_limiting.md:18]: @jimj-procoretech I completely understand why you are suggesting changes to tables.
To write tables in markdown means that each line of code could be super-long.

Still, this might be an indication that we should not use tables, but a different layout elements.

In fact [WWW Accessibility Rules](https://www.levelaccess.com/trouble-tables-brief-introduction/) remind us that the purpose of the `<table>` tag is to display tabular data, not to provide a specific layout for the HTML.

--- 

In this specific document, for instance, the text says:

> There are three important response headers returned when making a request to the Procore API.

What comes next, then, might be better specified as a **list** rather than a **table**, e.g.:

```markdown
- `X-Rate-Limit-Limit`: the total number of requests per 60 minute window
- `X-Rate-Limit-Remaining`: the number of requests you are allowed to make in the current 60 minute window
- `X-Rate-Limit-Reset`: the Unix timestamp for when the next window begins
```

or could even just be written in plain text in the same paragraph.

---

Using HTML inside a Markdown file opens a can of works to the maintanability and readability of this project.

- The same code goes from 5 lines to 23 lines of code
- When a new column is added to the table, maintainers will have to remember to edit both the `<thead>` and the `<tbody>` at once
- Markdown helpers such as `{% link … %}` are not available inside the HTML block, requiring maintainers to ad custom HTML and losing the benefits of link validations

---

In short, I agree with you that there are some tables in the current Markdown code that "look weird", with very long lines of code. My suggestion would be to identify them and to change them to a different layout (plain paragraphs or lists), while keeping Markdown tables for tabular data. What do you think?

File: api_essentials/restful_api_concepts.md [hunk 2]
```diff
@@ -64,11 +81,10 @@ The format type, query parameters, and any other required fields are contained w
 }
 ```
 
-###Responses
+### Responses
```
Comment (claudioprocore) [inline api_essentials/restful_api_concepts.md:84]: @jimj-procoretech Thanks for catching this!

File: building_applications/building_apps_manage_collabs.md [hunk 3]
```diff
@@ -28,17 +26,11 @@ You can add members to your development team and assign them to appropriate role
...
-After they accept the email invite and join the team, they have access to the App on the Developer Portal with the permissions defined by their assigned role.
+An email is sent out to the new team member inviting them to join your development team. The list of team members is updated to include the new team member with a status of 'Pending Invitation'. After they accept the email invite and join the team, they have access to the App on the Developer Portal with the permissions defined by their assigned role.
 
 ## Additional Actions
 
 In addition to adding new members to your development team, as an Owner or Admin you can perform other actions to help manage collaboration on your project.
 
 - You can transfer ownership of the App to one of the other team members.
 - You can remove a member from your development team.
-
```
Comment (claudioprocore) [inline building_applications/building_apps_manage_collabs.md:41]: @jimj-procoretech Thanks for this!

File: getting_started/making_first_call.md [hunk 4]
```diff
@@ -134,31 +118,25 @@ You have successfully made your first call to the Procore API!
...
 
 ![example call]({{ site.baseurl }}/assets/guides/example-call.png)
 
 Let's break down this example call:
 
 - First, we set the HTTP action to GET.
 - We then enter the URL for the List Projects endpoint as `https://api.procore.com/rest/v1.0/projects?company_id=1234`. Note that we've used a fictitous company_id of '1234', so you will want to substitute your own valid `company_id` value.
-- The Authorization Type is set to 'Inherit auth from parent' because we have configured OAuth 2.0 in Postman at the collection level as described in [Generating OAuth 2.0 Tokens]({{ site.url }}{{ site.baseurl }}{% link tools/postman.md %}#postman-token-gen).
+- The Authorization Type is set to 'Inherit auth from parent' because we have configured OAuth 2.0 in Postman at the collection level as described in [Generating OAuth 2.0 Tokens]({{ site.url }}{{ site.baseurl }}{% link tools/postman.md %}#generating-oauth-20-tokens-in-postman).
```
Comment (claudioprocore) [inline getting_started/making_first_call.md:139]: @jimj-procoretech Ooops, thanks for this!

**Review by claudioprocore — COMMENTED** [review]
@jimj-procoretech thank you so much for reviewing all these pages!

I left some inline comments which I invite you to read.

Some changes you made are perfect and very easy to spot (e.g.: you found some typos, wrong links, extra blank lines at the end).

Other changes are a little hard to review because new lines were removed at the end of each sentence, so it becomes unclear what exactly has changed on each line.

Let me know if my inline feedback is helpful. Thanks!

File: api_essentials/api_security_overview.md [hunk 5]
```diff
@@ -44,18 +27,16 @@ A service account has an auto-generated @procore.com email address which is inac
...
 
 Here is a list of additional resources to help you learn more about OAuth 2.0 and Procore API Security.
 
 - [Introduction to OAuth 2.0]({{ site.url }}{{ site.baseurl }}{% link oauth/oauth_introduction.md %}).
-- [OAuth 2.0 Specification](https://tools.ietf.org/html/rfc6749)
-- [OAuth 2.0 Simplified](https://aaronparecki.com/oauth-2-simplified/)
-- [Choosing an OAuth 2.0 Grant Type]({{ site.url }}{{ site.baseurl }}{% link oauth/oauth_choose_grant_type.md %})
-- [What is a Service Account?](https://support.procore.com/faq/what-is-a-service-account)
-- [Information Security and Privacy](https://support.procore.com/references/information-security-and-privacy)
+- [OAuth 2.0 Specification](https://tools.ietf.org/html/rfc6749){:target="_blank" rel="noopener"}
```
[the remark this answers is not in the document]
Comment (jimj-procoretech) [reply api_essentials/api_security_overview.md]: Sounds good. I added a new commit with the change.

File: api_essentials/rate_limiting.md (hunk 1 above)
Comment (jimj-procoretech) [reply api_essentials/rate_limiting.md:18]: That sounds good. Can we do re-evaluation and fixing of the tables once we've cut over to the new system?

Comment (claudioprocore) [reply api_essentials/rate_limiting.md:18]: @jimj-procoretech Yes we can! 

Would you like me to pull the remaining changes from this PR into a separate one, so we can apply those right away? I'm talking about the typos you find, the markdown mistakes, the extra blank lines at the end of some files…

File: api_essentials/api_security_overview.md [hunk 6]
```diff
@@ -8,34 +8,17 @@ section_title: API Essentials
...
-The OAuth 2.0 grant types supported by the Procore API rely on the use of tokens which are string values that represent the authorization and authentication of a specific application to access data in Procore on behalf of a Procore user.
+Procore employs what many consider the industry standard for API authentication - [OAuth 2.0](https://tools.ietf.org/html/rfc6749){:target="_blank" rel="noopener"}. The OAuth 2.0 authentication framework provides a secure means of authorizing and authenticating access to user data for third-party applications. Developers building integrations with the Procore API implement one of several [OAuth 2.0 authorization grant types]({{ site.url }}{{ site.baseurl }}{% link oauth/oauth_choose_grant_type.md %}) depending on their particular application use case. The OAuth 2.0 grant types supported by the Procore API rely on the use of tokens which are string values that represent the authorization and authentication of a specific application to access data in Procore on behalf of a Procore user.
```
[the remark this answers is not in the document]
Comment (jimj-procoretech) [reply api_essentials/api_security_overview.md]: Sounds good. So going forward the standard/guideline would be to have a new line the end of each sentence?

File: api_essentials/rate_limiting.md (hunk 1 above)
Comment (jimj-procoretech) [reply api_essentials/rate_limiting.md:18]: Sure sounds great!

File: api_essentials/api_security_overview.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (claudioprocore) [reply api_essentials/api_security_overview.md]: I would say "guideline" more than "standard".
It's not something that we have to enforce.
I just think it makes it easier to read, write, and review the code.

File: api_essentials/pagination.md [hunk 7]
```diff
@@ -23,7 +21,8 @@ Use the `per_page` parameter to specify how many items you want each page to ret
 Below is an example of using the `per_page` and page parameters in a typical API call.
 
 ```
-curl -i -H "Authorization: Bearer <your_authorization_code>" https://api.procore.com/rest/v1.0/punch_items?project_id=14406&per_page=5&page=3
+curl -i -H "Authorization: Bearer <your_authorization_code>" \
```
[the remark this answers is not in the document]
Comment (jimj-procoretech) [reply api_essentials/pagination.md:24]: We use 900px on the support site I think.
